examples/drivenet/drivenet_open_loop.ipynb

```diff
@@ -0,0 +1,348 @@
+    "model = torch.load(model_path).to(device)\n",
+    "model = model.eval()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Load the Evaluation Data\n",
+    "This is almost the exact same code you've already seen in the [training notebook](./drivenet_train.ipynb). Apart from the different dataset we load, the biggest difference is that **we don't perturb our data here**.\n",
+    "\n",
+    "When performing evaluation we're interested in knowing the performance on the annotated data, not on perturbed ones."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# ===== INIT DATASET\n",
+    "eval_cfg = cfg[\"val_data_loader\"]\n",
+    "rasterizer = build_rasterizer(cfg, dm)\n",
+    "eval_zarr = ChunkedDataset(dm.require(eval_cfg[\"key\"])).open()\n",
+    "eval_dataset = EgoDataset(cfg, eval_zarr, rasterizer)\n",
+    "eval_dataloader = DataLoader(eval_dataset, shuffle=eval_cfg[\"shuffle\"], batch_size=eval_cfg[\"batch_size\"], \n",
+    "                             num_workers=eval_cfg[\"num_workers\"])\n",
+    "print(eval_dataset)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# ==== EVAL LOOP\n",
+    "position_preds = []\n",
+    "yaw_preds = []\n",
+    "\n",
+    "position_gts = []\n",
+    "yaw_gts = []\n",
+    "\n",
+    "torch.set_grad_enabled(False)\n",
+    "idx = 0\n",
+    "\n",
+    "for data in tqdm(eval_dataloader):\n",
+    "    del data[\"host_id\"]\n",
+    "    result = model(data)\n",
+    "    position_preds.append(result[\"positions\"].detach().cpu().numpy())\n",
+    "    yaw_preds.append(result[\"yaws\"].detach().cpu().numpy())\n",
+    "\n",
+    "    position_gts.append(data[\"target_positions\"].detach().cpu().numpy())\n",
+    "    yaw_gts.append(data[\"target_yaws\"].detach().cpu().numpy())\n",
+    "    idx += 1\n",
+    "    if idx == 10:\n",
+    "        break\n",
+    "    \n",
+    "position_preds = np.concatenate(position_preds)\n",
+    "yaw_preds = np.concatenate(yaw_preds)\n",
+    "\n",
+    "position_gts = np.concatenate(position_gts)\n",
+    "yaw_gts = np.concatenate(yaw_gts)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Quantitative Evaluation: ADE, FDE and Angle Distance\n",
+    "\n",
+    "#### Positional Displacement\n",
+    "Average Displacement Error (ADE) and Final Displacement Error(FDE) are standard metrics used to evaluate future predictions for AVs.\n",
+    "\n",
+    "We can easily compute them by comparing predicted and annotated positions, which we have saved in the previous cell.\n",
+    "Additionally, we can plot histograms of their distributions across samples to better capture the variance of our error\n",
+    "\n",
+    "#### Angle Displacement\n",
```

Comment:
Why is this useful? Bit less obvious to me than position - or rather, ADE and FDE are better motivated.

Reply:
my intuition is that if we don't include it we're not reporting any metrics on the yaw. In close loop later yaw is important for predictions, so it makes sense to check if it's reasonable imo